partcad_skyline/skyline.py

```
@@ -0,0 +1,146 @@
+                if author and commit.author.email != author:
+                    continue
+                commit_date = commit.committed_datetime.date().isoformat()
+                if commit_date in commit_counts:
+                    commit_counts[commit_date] += 1
+                else:
+                    commit_counts[commit_date] = 1
+            total_days_with_commits = len(commit_counts)
+            total_commits = sum(commit_counts.values())
+            logging.info(f"Total days with commits: {total_days_with_commits}")
+            logging.info(f"Total number of commits: {total_commits}")
+            return commit_counts
```

Ruff (0.8.0)

Consider moving this statement to an `else` block

(TRY300)